core/src/main/scripts/importer/validateData.py

```diff
@@ -2043,6 +2045,13 @@ def checkLine(self, data):
                         extra={'line_number': self.line_number,
                                'column_number': col_index + 1,
                                'cause': value})
+                if re.findall(self.INVALID_SAMPLE_ID_CHARACTERS, value):
+                    self.logger.error(
+                        'PATIENT_ID can only contain letters, numbers, points, '
+                        'underscores and hyphens',
+                        extra={'line_number': self.line_number,
+                               'column_number': col_index + 1,
+                               'cause': value})
```

Comment:
As the validation for PATIENT_ID is the same as for SAMPLE_ID, could you move this check to the super class `ClinicalValidator`?

Reply:
Also, the "White space in PATIENT_ID is not supported" error is now captured by the new check. Can we remove this redundant check for both PATIENT_ID and SAMPLE_ID?